pkg/yang/types_builtin.go

```diff
@@ -944,10 +945,9 @@ func (r YangRange) Less(i, j int) bool {
 	}
 }
 
-// Validate sorts r and returns an error if r has either an invalid range or has
+// Validate returns an error if r has either an invalid range or has
 // overlapping ranges.
```

Comment:
Maybe just add at the end of the comment that it expects the range to be sorted.
Maybe also add to the docstring for `parseRanges`.

Reply:
Added.